css/index.css

```diff
@@ -110,6 +124,108 @@ p {
     display: flex;
     align-items: center;
     justify-content: center;
+}*/
+
```

Comment:
On these boxes, the parent can take care of everything except for the one override each box needs, the color.